core/arch/arm/plat-vexpress/platform_config.h

```diff
@@ -237,13 +242,19 @@
 #define DRAM0_TEERES_BASE	(DRAM0_BASE + DRAM0_SIZE)
 #define DRAM0_TEERES_SIZE	CFG_SHMEM_SIZE
 
-/* Secure RAM: SDP memory followed by OP-TEE secure memory */
-#define SECRAM_BASE		0x0e000000
-#define SECRAM_SIZE		0x01000000
+/*
+ * Secure RAM:     SDP disabled    |  SDP enabled
+ *   ARM-TF        1M@0x0e000000   |   1M@9x0e000000
+ *   SDP memory        n.a         |   2M@0x0e100000
```

Comment:
The qemu bootloader curerntly loads the OP-TEE formatted image at end of the secure RAM. If the OP-TEE pager is enable, then OP-TEE core will copy the pagestore from its initial location (end of secure RAM) into an allocated location in TA RAM. This copy is done when MMU is enable. That said, remember that SDP memory pool(s) are not default mapped to OP-TEE code. Hence, if the end of the secure RAM stores the pagestore, OPTEE core will not be able to read it when copying it.

A simple solution, is to have a bootloader aware that end of secure RAM is not OP-TEE RAM but SDP RAM, so bootloader shall be modified so that it loads the OPTEE image somewhere in the OP-TEE RAM (i.e end TA RAM), not in the SDP RAM.